pyproject.toml

```
@@ -37,7 +37,7 @@ file = "LICENSE"
 "lclsspeak" = "lclsspeak.bin.main:main"
 
 [tool.setuptools_scm]
-write_to = "lclsspeak/_version.py"
+version_file = "lclsspeak/_version.py"
```

```suggestion
write_to = "lclsspeak/_version.py"
```

we pinned the ci-helpers to setuptools_scm < 8.0, so we use the old `write_to`.